lib/emails/invitation.ts

```diff
@@ -26,28 +28,30 @@ const sendEmail = (invitation: Invitation, provider: EmailProvider): Promise<voi
   new Promise((resolve, reject) => {
     const { transport, from } = provider;
 
+    const { language: t } = invitation;
     const invitationHtml = html(invitation);
     nodemailer.createTransport(transport).sendMail(
       {
         from: `Cal.com <${from}>`,
         to: invitation.toEmail,
-        subject:
-          (invitation.from ? invitation.from + " invited you" : "You have been invited") +
-          ` to join ${invitation.teamName}`,
+        subject: invitation.from
+          ? t("user_invited_you", { user: invitation.from, teamName: invitation.teamName })
+          : t("you_have_been_invited", { teamName: invitation.teamName }),
         html: invitationHtml,
         text: text(invitationHtml),
       },
       (error) => {
         if (error) {
```

Comment:
`if (error instanceof Error)` is better here.

Reply:
Does `if (_err) ` necessary? Can it be undefined or null?